frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -533,6 +533,28 @@ describe("scenarios > dashboard", () => {
       cy.findAllByText("Bobby Tables");
       cy.contains(/revert/i);
     });
+
+    it("should show sub-day resolutions in relative date filter (metabase#6660)", () => {
+      cy.visit("/dashboard/1");
+      cy.get(".Icon-pencil").click();
+      cy.get(".Icon-filter").click();
+      popover().within(() => {
+        cy.findByText("Time").click();
+        cy.findByText("All Options").click();
+      });
+      cy.findByText("No default").click();
+      // click on Previous, to expand the relative date filter type dropdown
+      cy.findByText("Previous").click();
+      // choose Next, under which the new options should be available
+      cy.findByText("Next").click();
+      // click on Days, which should open the resolution dropdown
+      cy.findByText("Days").click();
+      // Minutes should appear in the selection box; click it
+      cy.findByText("Minutes").click();
+      // also check the "Include this minute" checkbox
+      // which is actually "Include" followed by "this minute" wrapped in <strong>, so has to be clicked this way
+      cy.findByText("this minute").click();
```

Comment:
When you encounter text broken into multiple elements, you can use:
- `cy.contains("this minute")`
    - be careful with it because it can match many things (for example: "Include this minute" and "This minute feels like an hour" - in some hypothetical scenario, of course)
- `cy.findByText(/this minute$/)` is probably even more specific

Reply:
Updated the test as per our discussion.

Reply:
Awesome!